modin/engines/base/frame/data.py

```diff
@@ -1056,12 +1056,17 @@ def _map_reduce_func(df, *args, **kwargs):
                 # axis=1 does not have this requirement because the index already will
                 # line up with the index of the data based on how pandas creates a
                 # DataFrame from a Series.
-                return pandas.DataFrame(series_result).T
-            return pandas.DataFrame(series_result)
+                result = pandas.DataFrame(series_result).T
+                result.index = ["__reduced__"]
+            else:
+                result = pandas.DataFrame(series_result)
+                if isinstance(series_result, pandas.Series):
```

Comment:
Is this for the case where the map reduce doesn't fully reduce down to a single value per row? (since this is a column-wise reduction).

Reply:
Yes, the result does not have a name, so we give it a name that is shared externally and internally.

Reply:
Is there a case where series_result is not a pandas.Series? What would it be otherwise?

Reply:
It might not always be a `Series`, it could be a dataframe. We always convert it to a dataframe here in order to treat them correctly because `Series` will not always have a name.